Common/UnitDefinitions/MassConcentration.json

```diff
@@ -0,0 +1,246 @@
+      "Prefixes": [ "Kilo", "Milli", "Micro" ],
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "g/m³" ]
+        },
+        {
+          "Culture": "ru-RU",
+          "Abbreviations": [ "г/м³" ]
+        }
+      ]
+    },
+    {
+      "SingularName": "PoundPerCubicInch",
+      "PluralName": "PoundsPerCubicInch",
+      "BaseUnits": {
+        "L": "Inch",
+        "M": "Pound"
+      },
+      "FromUnitToBaseFunc": "x/3.6127298147753e-5",
+      "FromBaseToUnitFunc": "x*3.6127298147753e-5",
+      "Prefixes": [ "Kilo" ],
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "lb/in³" ],
+          "AbbreviationsWithPrefixes": [ "kip/in³" ]
+        }
+      ]
+    },
+    {
+      "SingularName": "PoundPerCubicFoot",
+      "PluralName": "PoundsPerCubicFoot",
+      "BaseUnits": {
+        "L": "Foot",
+        "M": "Pound"
+      },
+      "FromUnitToBaseFunc": "x/0.062427961",
+      "FromBaseToUnitFunc": "x*0.062427961",
+      "Prefixes": [ "Kilo" ],
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "lb/ft³" ],
+          "AbbreviationsWithPrefixes": [ "kip/ft³" ]
+        }
+      ]
+    },
+    {
+      "SingularName": "TonnePerCubicMillimeter",
+      "PluralName": "TonnesPerCubicMillimeter",
+      "BaseUnits": {
+        "L": "Millimeter",
+        "M": "Tonne"
+      },
+      "FromUnitToBaseFunc": "x/1e-12",
+      "FromBaseToUnitFunc": "x*1e-12",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "t/mm³" ]
+        }
+      ]
+    },
+    {
+      "SingularName": "TonnePerCubicCentimeter",
+      "PluralName": "TonnesPerCubicCentimeter",
+      "BaseUnits": {
+        "L": "Centimeter",
+        "M": "Tonne"
+      },
+      "FromUnitToBaseFunc": "x/1e-9",
+      "FromBaseToUnitFunc": "x*1e-9",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "t/cm³" ]
+        }
+      ]
+    },
+    {
+      "SingularName": "TonnePerCubicMeter",
+      "PluralName": "TonnesPerCubicMeter",
+      "BaseUnits": {
+        "L": "Meter",
+        "M": "Tonne"
+      },
+      "FromUnitToBaseFunc": "x/0.001",
+      "FromBaseToUnitFunc": "x*0.001",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "t/m³" ]
+        }
+      ]
+    },
+    {
+      "SingularName": "SlugPerCubicFoot",
+      "PluralName": "SlugsPerCubicFoot",
+      "BaseUnits": {
+        "L": "Foot",
+        "M": "Slug"
+      },
+      "FromUnitToBaseFunc": "x*515.378818",
+      "FromBaseToUnitFunc": "x*0.00194032033",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "slug/ft³" ]
+        }
+      ]
+    },
+    {
+      "SingularName": "GramPerLiter",
+      "PluralName": "GramsPerLiter",
+      "FromUnitToBaseFunc": "x/1",
```

Comment:
Missing `"BaseUnits"` here and several below. 

For your information, we recently introduced this concept and haven't added it to all our existing units yet, so I assume you copied some existing definitions that didn't have them.

Note that not all units make sense to define BaseUnits for, such as dimensionless units or units with no obvious mapping to SI base units.

Reply:
I was indeed copying over from the Density.. I've updated MassConcentration & Molarity types to use Kilogram/Meter & Mole/Meter respectively (incl ppg?). However I am still a little unclear as to what I should prefer when selecting between equivalent base units? I presume there is a difference in precision? So should I aim in the higher or lower range?  Could you give me an example?

Reply:
Good questions, we are kind of treading the path as we go on this new `UnitSystem` and `BaseUnits` concept so I don't have all the answers yet.

1. The choice of `BaseUnits` does not affect precision, you are maybe confused with `BaseUnit` on the quantity as opposed to `BaseUnits` on the unit? 

In https://github.com/angularsen/UnitsNet/issues/563#issuecomment-467029946 I propose renaming `Length.BaseUnit` to `Length.DefaultUnit` to avoid this ambiguity.

I updated the [wiki's step 1](https://github.com/angularsen/UnitsNet/wiki/Adding-a-New-Unit#1-add-or-modify-json-file-for-unit-class) to be a bit more clear about `BaseUnit` vs `BaseUnits`, so please give that another read and let me know if that helps clear things up?

2. As for two equivalent units, they should specify the same `BaseUnits`. One of them (first match) will be selected when constructing quantities using `UnitSystem` argument. It has no effect if not using `UnitSystem` to construct quantities, as I have now tried to clarify in the wiki.

3. I see your latest commit updated `BaseUnits` from say `Gram` to `Kilogram`, but it was originally correct. Please see the examples in the linked wiki and let me know if that helps or should be further improved.